pytest_pylint.py

```diff
@@ -111,6 +111,11 @@ class PyLintItem(pytest.Item, pytest.File):
     def __init__(self, fspath, parent, msg_format=None, pylintrc_file=None):
         super(PyLintItem, self).__init__(fspath, parent)
 
+        if hasattr(self, 'add_marker'):
```

Comment:
This is great, but why can't it just be `self.add_marker("pylint")` like it is in pytest-pep8 https://bitbucket.org/pytest-dev/pytest-pep8/src/44ddb3739f0f892fc0029d5d809496ec5ef24f28/pytest_pep8.py?fileviewer=file-view-default#pytest_pep8.py-56 ?